### PR TITLE
Enable the user to see the current configuration from the UI

### PR DESCRIPTION
**Pull request by Pols95**
This is just the first step of enabling user to see current config (#10 ).
What about merging what done up to now (so we can go ahead testing easier) and then develop later (asap) the remaining parts? @eolivelli

### CODE REVIEW

File: carapace-ui/src/main/webapp/public/index.html [hunk 1]
```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
     <title>Http Proxy UI</title>
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
```
Comment (eolivelli) [inline carapace-ui/src/main/webapp/public/index.html]: Better not to use cdn but copy the contents, please check locense

File: carapace-ui/src/main/webapp/src/components/Configuration.vue [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+            this.reload()
+        },
+        methods: {
+            save: function () {
+                var self = this
+                axios.post('/api/config/apply', this.configuration, this.axiosParams).then(function (response) {
+                    var data = response.data
+                    self.opSuccess = data.ok
+                    self.opMessage = data.ok ? "Configuration saved successfully." : "Error on configuration saving: " + data.error
+                    console.log(response)
```
Comment (eolivelli) [inline carapace-ui/src/main/webapp/src/components/Configuration.vue]: Remove debug

File: carapace-ui/src/main/webapp/src/components/Configuration.vue [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                    self.opSuccess = false
+                    self.opMessage = "Error on reloading current configuration:" + error
+                    console.log(error)
+                })
+            }
+        }
+    }
+</script>
+
+<style>
```
Comment (eolivelli) [inline carapace-ui/src/main/webapp/src/components/Configuration.vue]: Is this the right place? Don't we have some common place for css?

Comment (Pols95) [reply carapace-ui/src/main/webapp/src/components/Configuration.vue]: I changed it as component-scoped style

**Comment by eolivelli** [comment]
@nicoloboschi  please take another look

**Comment by nicoloboschi** [comment]
Look good to me (we should rename mockserver javascript file but we can do later)

**Pull request merged**